test/lint.js

```diff
@@ -0,0 +1,145 @@
+'use strict';
+
+let assert = require('assert');
+let emu = require('../lib/ecmarkup');
+
+function alg([before, after], marker) {
+  return `<emu-alg>${before}${marker}${after}</emu-alg>`;
```

Comment:
I don't see why we actually include the marker. We just need to track positions here, right?

Reply:
We could thread location information through, but this is easier and I am not inclined to do the fancier thing just for the tests.

Reply:
Yeah but this strategy won't work for all algorithm texts and marker positions. I'd prefer we do the right thing now so that later PRs don't have to both change up this testing strategy and implement a linting rule. I can pair with you on it if you want, it shouldn't take long.

Reply:
It's not hard, I just don't think it's worth doing. Feel free to send a followup PR.

Reply:
I did it.